content/docs/atplatform/specification.md

```diff
@@ -201,6 +201,26 @@ A user of the atServer should be able to decide who is allowed to connect to the
 
 ## Verbs
 
+Verbs are commands you can execute on a secondary server. Each verb interacts with the secondary server in a different way. Some are for authentication, some are for data retrieval and some are for data manipulation. See the table below for more insight on which verb to use for what purpose.
+
+| Verb | Purpose |
```

Comment:
Can we move `update` and `update:meta` down so that things are sorted by purpose? And also, does it make sense to do so.